src/app/components/app-node-utilizations/app-node-utilizations.component.scss

```diff
@@ -16,13 +16,22 @@
  * limitations under the License.
  */
 
- .app-node-utilizations {
+::ng-deep .app-node-utilizations {
+    .card-wrapper {
+        max-width: 800px;
```

Comment:
I don't think we should have a max-width here. With lots of resource types, the extra horizontal real estate can be nice to have.

Reply:
This also means the chart should expand to fill the available space.

Reply:
Okay. I have removed the restrict, the updated demo shown as above.
The reason why I apend the restrict is consider the right side partition selector might be far for user.